alembic/versions/9152eefe06e0_create_initial_application_table.py

```diff
@@ -0,0 +1,36 @@
+"""Create initial application table
+
+Revision ID: 9152eefe06e0
+Revises:
+Create Date: 2023-08-09 12:54:19.543072
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSON, UUID
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "9152eefe06e0"
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "application",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("claim_id", UUID(as_uuid=True), nullable=False),
+        sa.Column("version", sa.Integer, nullable=False),
```

Comment:
do we want to either have a latest flag to avoid having to calculate that on the fly or use a reference field to make the versions a linked list (i.e. previous_version_id)

Reply:
First thought was this is as dumb as possible to reduce any logic in the actual service. This could be potentially something we add in the future though